packages/perseus/src/widgets/numeric-input/input-with-examples.tsx

```diff
@@ -45,7 +45,7 @@ type Props = {
  * examples of how to input the selected answer forms.
  */
 const InputWithExamples = forwardRef<Focusable, Props>(
-    (
+    function InputWithExamples(
```

Comment:
Another change so that this component shows up nicely named in React dev tools (and stack traces)